web/src/composebox_typeahead.ts

```diff
@@ -936,17 +924,48 @@ export function get_candidates(
                 }
 
                 const stream_id = stream_data.get_stream_id(stream_name);
+                // We always show topic suggestions after the user types a stream,
+                // and let them pick between just showing the stream (the first option, when nothing follows ">")
+                // or adding a topic.
```

Comment:
These comments should probably be line-wrapped a bit more tightly.

We allow up to 100 characters, but prefer to wrap comments to ~80 for readability; usually very long lines are for URLs and the like.

Reply:
Is this something we could add a linter rule for?

Reply:
It's hard to configure the linter to do that. We use a third-party linter for this detail, and it only supports a hard maximum.